### DISCUSSION TITLE
Iniciando cadastro de usuários - Autenticação e autorização (Fundamento do ASP.NET 6)

### DISCUSSION THREAD

**Discussion by BernardoMeine** [balta-io/forum#522] [Tira Dúvidas] [answered]
Na hora que o balta altera o UserMap e faz as mudanças nas propiedades, o HasMaxLenght tanto do Email quanto do Slug não deveriam ser iguais tendo em vista que o Slug do usuário vai ser o email com os caracteres "@" e "." substituídos por "-"?
Da forma que o UserMap ficou, vai haver problemas na criação de um Slug para um usuário com email maior que 80 caracteres.

**Comment by andrebaltieri** [comment, marked as answer]
Bom dia, @BernardoMeine, bem observado... se vai seguir com E-mail como username, precisa sim!
Obrigado 🚀
